src/main/java/core/basesyntax/service/operation/ReturnOperationHandler.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.service.operation;
+
+import core.basesyntax.service.FruitTransaction;
+import java.util.Map;
+
+public class ReturnOperationHandler implements OperationHandler {
```

Comment:
According to SOLID principles, each class should have a single responsibility. This class is responsible for handling the 'return' operation. However, it's not clear what 'return' operation implies. Consider renaming this class to make its purpose more clear, for example 'ReturnFruitOperationHandler'. This will make your code more readable and maintainable.